src/execution/execute.js

```diff
@@ -862,6 +906,24 @@ function completeListValue(
   path: Path,
   result: mixed,
 ): PromiseOrValue<$ReadOnlyArray<mixed>> {
+  const itemType = returnType.ofType;
+
+  if (isAsyncIterable(result)) {
+    const iteratorMethod = result[SYMBOL_ASYNC_ITERATOR];
+    const iterator = iteratorMethod.call(result);
```

Comment:
@robrichard Can we do?
```suggestion
    const iterator = result[SYMBOL_ASYNC_ITERATOR](result);
```

Reply:
Yes, updated.